packages/editor/src/plugins/fill-in-the-blanks-exercise/components/blank-draggable-dummy-answer.tsx

```diff
@@ -0,0 +1,53 @@
+import { EditorTooltip } from '@editor/editor-ui/editor-tooltip'
+import { faCircleXmark } from '@fortawesome/free-solid-svg-icons'
+import {
+  KeyboardEventHandler,
+  type ChangeEventHandler,
+  type MouseEventHandler,
+} from 'react'
+
+import { FaIcon } from '@/components/fa-icon'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+import { cn } from '@/helper/cn'
+
+interface BlankDraggableDummyAnswerProps {
+  text: string
+  onChange: ChangeEventHandler<HTMLInputElement>
+  onKeyDown: KeyboardEventHandler
+  onRemoveClick: MouseEventHandler
+}
+
+export function BlankDraggableDummyAnswer(
```

Comment:
Maybe rename to `ExtraDragAndDropAnswer`? By the name I would have expected it to be a draggable element mixed with the other draggable elements.